### ISSUE TITLE
borg-macosx64 opens in text editor

### ISSUE THREAD

**Issue by CouldBeThis** [borgbackup/borg#5517]
## Have you checked borgbackup docs, FAQ, and open Github issues?

yes

## Is this a BUG / ISSUE report or a QUESTION?

?issue?

#### Operating system (distribution) and version.
mac os 10.1416

## Describe the problem you're observing.

I am unable to use the file https://github.com/borgbackup/borg/releases/download/1.1.14/borg-macosx64. it displays with a plain document (unknown file type) icon and opens in a text editor. I tried appending .dmg to the filename but it wouldn't mount. 

some sort of mistake in creating the file I guess.

**Comment by ThomasWaldmann** [comment]
It is a binary executable file, but after download you maybe need to set the executable bit:

```
chmod +x borg-macosx64
```

**Comment by ThomasWaldmann** [comment]
so, did that help?

**Comment by CouldBeThis** [comment]
Hi,

I did that and was able to run this fine. Thank you!

However clicking around in the docs I can't remember ever reading any of the information... it is plain from the [instructions](https://borgbackup.readthedocs.io/en/1.1-maint/installation.html#standalone-binary) that the way I was trying to open this would never have worked. _Obviously_ it would not be a dmg because cli apps are never dmgs. 

So I think that I must have been confused when I grabbed this initially. maybe I was looking at 2 things at the same time. 

look like I was wasting your time! apologies.
